### PR TITLE
Fixed Broken Instagram Images

### LINKED ISSUE #2297: Blank Instagram preview
## Expected Behavior
https://www.thebluealliance.com/team/6090 shows a blank preview for https://www.instagram.com/p/BgH9kCEhFUU/

## Current Behavior
image

Photo is not blank

image

### PR DESCRIPTION
**Pull request by dracco1993**
## Description
Move Instagram image presentation to use the suggested (by Instagram) media embedding URLs.

## Motivation and Context
Some Instagram images are broken, due to us saving media URLs that are now outdated.
This fixes #2297.

## How Has This Been Tested?
Locally.

## Screenshots (if appropriate):
The second image in the screenshot is from Instagram:

And the image carousel, with said Instagram image:

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change API specifications or require data migrations)

### CODE REVIEW

**Comment by dracco1993** [comment]
While this _is_ working everywhere for the initial image rendering, the `302` redirect that Instagram is doing in the background for the `jpg` is breaking the image carousel we're using, so I still need to figure that out.

**Comment by dracco1993** [comment]
Alright, pending tests, this should be good to review/test locally then merge.

File: models/media.py [hunk 1]
```diff
@@ -145,15 +145,18 @@ def instagram_url(self):
...
     def instagram_direct_url_med(self):
-        return self.instagram_direct_url
+        return self.instagram_url_helper("m")
 
     @property
     def instagram_direct_url_sm(self):
-        return self.instagram_direct_url
+        return self.instagram_url_helper("t")
+
+    def instagram_url_helper(self, size):
```
[the remark this answers is not in the document]
Comment (dracco1993) [reply models/media.py]: Sure.  Added in bb084b8

**Comment by sorianog** [comment]
This is awesome. Thanks for your work!

**Pull request merged**